contracts/Tokens/VTokens/VToken.sol

```diff
@@ -500,6 +500,7 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
                 );
         }
 
+        uint availableReserves = totalReservesNew < cashPrior ? totalReservesNew : cashPrior;
```

Comment:
We could move this to the `if` statement where the var `availableReserves` is used. This way we would save some gas most of the cases